jQuery/honeyHeadquarters.js

```diff
@@ -1,25 +1,25 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
 $.ajax('http://localhost:3000/citizens').done(function(citizens) {
```

Comment:
not really sure I understand what you mean here. Move the ENTIRE block into another file, or just the function declarations? the latter makes more sense to me, since then I can reuse the functions elsewhere which addresses the problem you brought up, and I know that the last six lines of the ajax block are only relevant there, but it seems like you're suggesting an alternative